components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc

```diff
@@ -379,8 +379,8 @@ void CosmeticFiltersJSHandler::ApplyRules() {
     return;
 
   // Working on css rules, we do that on a main frame only
-  bool generichide = false;
-  resources_dict_->GetBoolean("generichide", &generichide);
+  bool generichide =
+      resources_dict_->FindBoolPath("generichide").value_or(false);
```

Comment:
`FindBoolKey`

Reply:
Fixed in 2713c0a45d9d84f7cb4a310c982e48a7064bd900